invenio_app_rdm/records_ui/views/records.py

```diff
@@ -203,6 +203,7 @@ def record_detail(pid_value, record, files, is_preview=False):
         is_draft=is_draft,
         stats=_get_stats(record_ui["id"], record_ui["parent"]["id"]),
         community=resolved_community,
+        external_resources=external_resources(record_ui),
```

Comment:
you need to add it to the context of request views as well, they use the record template